packages/camera/camera/lib/src/camera_image.dart

```diff
@@ -90,6 +90,9 @@ ImageFormatGroup _asImageFormatGroup(dynamic rawFormat) {
       // android.graphics.ImageFormat.JPEG
       case 256:
         return ImageFormatGroup.jpeg;
+      // android.graphics.ImageFormat.NV21
+      case 17:
+        return ImageFormatGroup.nv21;
```

Comment:
This could also use a test. Probably in https://github.com/flutter/packages/blob/main/packages/camera/camera/test/camera_image_test.dart

Reply:
Apologies - I thought I had that one, maybe I forgot to include it when I combined the PRs into one.

Reply:
Fixed in 97787d5f5a619cd13a824e54ffccd4e55f346bb2

Reply:
Merged main and removed tracked gradle files in 14f29861ce64a0ef7bfa3da3d7d8953b95d0f0e7